docs/toc.yml

```diff
@@ -341,6 +341,8 @@ items:
       href: develop/usability-testing.md
     - name: Validate an add-in's manifest
       href: testing/troubleshoot-manifest.md
+    - name: Runtimes in Office Add-ins
+      href: testing/runtimes.md
```

Comment:
Should this be in the testing section? Maybe it could be in develop (e.g. under "Setup and run your add-in") or plan

Reply:
> Should this be in the testing section? Maybe it could be in develop (e.g. under "Setup and run your add-in") or plan

I've gone back and forth on that, (and I can see the case either way) but my conclusion is that devs don't need/want a deep dive about runtimes when they are planning. The place they are most likely to be when they find they'd like a deep dive on runtimes is when they are testing/debugging. Make sense?

Reply:
I see. ok.

Reply:
+1